src/OversetSimulation.h

```diff
@@ -109,6 +111,30 @@ class OversetSimulation
         m_is_adaptive_holemap_alg = alg;
         if (m_is_adaptive_holemap_alg == true) m_tg.setHoleMapAlgorithm(1);
     }
+
+    void set_composite_num(int num_composite)
```

Comment:
the argument to the function can be `const int`

Reply:
I addressed this.